src/static/bitbucket/components/Message.tsx

```diff
@@ -115,14 +122,33 @@ const Message = ({
   const showWarnings = status === 'can-land' || status === 'cannot-land' || status === 'queued';
   const showErrors = status === 'cannot-land' || status === 'queued';
 
+  const landButton = (
+    <div style={{ marginRight: 15 }}>
+      <Confetti
+        active={loading === 'land'}
+        config={{
+          angle: 20,
+          spread: 58,
+          startVelocity: 50,
+          elementCount: 70,
+          dragFriction: 0.12,
+          duration: 3000,
+          stagger: 3,
+          width: '10px',
+          height: '10px',
+          // Missing type
+          // @ts-ignore
+          perspective: '500px',
+        }}
+      />
+      <Button appearance="primary" onClick={onLandClicked} isLoading={loading === 'land'}>
+        Land changes
+      </Button>
+    </div>
+  );
+
   return (
-    <div
-      style={{
-        // Fixed height prevents cumulative layout shift caused by slow loads
-        height: 270,
```

Comment:
The fixed height prevents layout shift, as Landkid is usually the last thing to load and it pushes all other content down. The loading skeleton will help but still prone to cause jumps. I feel we shouldn't remove this unless we can drastically reduce Landkid's loading times to a point where it isn't as noticeable 🤔

Reply:
@luke-underwood The shimmer hopefully reduces the CLS to a level where it isn't a big concern.
Having a fixed height introduces a few problems —

#### White gap
<img width="1483" alt="Screenshot 2022-09-01 at 9 01 14 AM" src="https://user-images.githubusercontent.com/8946207/187826110-f35cf221-04c9-441b-abe5-107562be44fe.png">

#### Overlap (exaggerated by moving content inside section message)
<img width="1487" alt="Screenshot 2022-09-01 at 9 01 44 AM" src="https://user-images.githubusercontent.com/8946207/187826118-c5fa7292-28d9-4577-af28-35010a4645b3.png">

Reply:
>  I feel we shouldn't remove this unless we can drastically reduce Landkid's loading times to a point where it isn't as noticeable

Hopefully, the landkid status check times should be cut down by more than half after https://github.com/atlassian/landkid/pull/167 is in. Would still be noticeable though.

Reply:
Also, added a fixed height here —
[src/static/bitbucket/index.html](https://github.com/atlassian/landkid/pull/169/files#diff-42716c2f50292cfbd69d7319e1865e2164bddd9c947c3aa7bc98f1ab0d9e6bba)

which was causing the initial CLS

Reply:
> Hopefully, the landkid status check times should be cut down by more than half after https://github.com/atlassian/landkid/pull/167 is in

Wow that'd be great to see. In that case I'm happy to eat the layout shift for now 👍